src/FactorGraph.jl

```diff
@@ -392,84 +396,67 @@ addVariable!(fg, :x0, Pose2)
 ```
 """
 function addVariable!(dfg::AbstractDFG,
-                      lbl::Symbol,
-                      softtype::InferenceVariable;
+                      label::Symbol,
+                      varType::InferenceVariable;
                       N::Int=100,
-                      autoinit::Union{Nothing,Bool}=nothing,
                       solvable::Int=1,
                       timestamp::Union{DateTime,ZonedDateTime}=now(localzone()),
-                      nanosecondtime::Union{Nanosecond,Int64,Nothing}=nothing,
+                      nanosecondtime::Union{Nanosecond,Int64}=Nanosecond(0),
                       dontmargin::Bool=false,
                       labels::Union{Vector{Symbol},Nothing}=nothing,
                       tags::Vector{Symbol}=Symbol[],
                       smalldata=Dict{Symbol, DFG.SmallDataTypes}(),
                       checkduplicates::Bool=true,
-                      initsolvekeys::Vector{Symbol}=getSolverParams(dfg).algorithms)::DFGVariable
+                      initsolvekeys::Vector{Symbol}=getSolverParams(dfg).algorithms )
   #
-
-  if :ut in fieldnames(typeof(softtype))
-    Base.depwarn("Field `ut` (microseconds) for softtype $(softtype) has been deprecated please use DFGVariable.nstime, kwarg: nanosecondtime", :addVariable!)
-    if isnothing(nanosecondtime)
-      nanosecondtime = Nanosecond(softtype.ut*1000)
-    else 
-      @warn "Nanosecond time has been specified as $nanosecondtime, ignoring `ut` field value: $(softtype.ut)."
-    end
-  elseif isnothing(nanosecondtime)
-    nanosecondtime = Nanosecond(0)
-  end
```

Comment:
@dehann do you have a reason for deleting this? Or can I put it back?

Reply:
oh sorry, that was probably an accident!  I was clearing out lots of really old code and this was probably taken with.  We can put it back.